CMakeLists.txt

```diff
@@ -111,6 +111,14 @@ if(TYPE_SAFE_IMPORT_STD_MODULE)
     # If a user has set TYPE_SAFE_IMPORT_STD_MODULE to ON this indicates they want the type_safe headers to import std
     # This is a C++ 23 feature so we require them to use C++ in their build
     target_compile_features(type_safe INTERFACE cxx_std_23)
+
+    # Hopefully MS will provide a better/automatic/IJW way to use the standard library module, in other words "import std;" should just work
+    if( MSVC )
+        # We use a path relative to the compiler location which should be stable and avoid VS upgrade/version/preview install location issues
+        cmake_path(GET CMAKE_CXX_COMPILER PARENT_PATH  CompilerDir)
+        target_sources(type_safe PUBLIC "${CompilerDir}/../../../modules/std.ixx" )
```

Comment:
Done

Reply:
Note that reverting the changes means that any program, like the exmaple project will have to replicate the stuff to add the ixx file. That was why I added it to the type_safe project so that users would get that automatically